vms/platformvm/txs/executor/proposal_tx_executor.go

```diff
@@ -320,215 +322,303 @@ func (e *ProposalTxExecutor) RewardValidatorTx(tx *txs.RewardValidatorTx) error
 	if !currentStakerIterator.Next() {
 		return fmt.Errorf("failed to get next staker to remove: %w", database.ErrNotFound)
 	}
-	stakerToRemove := currentStakerIterator.Value()
+	stakerToReward := currentStakerIterator.Value()
```

Comment:
before continuos staking this staker is both rewarded and removed. After continuous staking, staker may be restaked and it won't be necessarily removed. Just a nit, but it'll reduce diffs down the line